http3/config.py

```diff
@@ -185,6 +215,8 @@ class PoolLimits:
     Limits on the number of connections in a connection pool.
     """
 
+    __slots__ = ("soft_limit", "hard_limit", "pool_timeout")
```

Comment:
I think let's pass on making `__slots__` changes in PRs.

If we do want to add them in then let's treat em as an independent PR. (Tho I'd be tempted to say we shoud simply ignore them, at least until we're closing in on a 1.0??)

Reply:
Sure! I'll revert these.